iree/turbine/kernel/wave/cache.py

```diff
@@ -166,7 +166,7 @@ def get_hash(
 
         # Benchmark related hash
         if run_bench and config != None:
-            key += config.get("benchmark_batch_size", "")
+            key += [config.get("benchmark_batch_size", "")]
```

Comment:
Do we need this change?

Reply:
Yeah, otherwise there's an issue when an `int` is being subscripted

Reply:
makes sense, thanks!